test/helpers/run-webpack-dev-server.js

```diff
@@ -31,7 +31,7 @@ function runWebackDevServer(testArgs, configPath) {
   const args = [webpackDevServerPath, '--config', configPath].concat(testArgs);
 
   return new Promise((resolve, reject) => {
-    const child = spawn('node', args, { cwd, env });
+    const child = execa('node', args, { cwd, env });
```

Comment:
It is strange to use `spawn`, the following error occurs.

```
 spawn node ENOENT
```

So We will use execa.